addons/project_timesheet_holidays/models/hr_holidays.py

```diff
@@ -73,7 +74,7 @@ def _validate_leave_request(self):
                 fields.Datetime.from_string(holiday.date_to),
             )
             for index, (day_date, work_hours_count) in enumerate(work_hours_data):
-                self.env['account.analytic.line'].sudo().create({
+                analytic_line_vals.append({
```

Comment:
1/ we are talking of timesheet here, so avoid `analytic_line_` prefix ;)
2/ you probably need to rebase this, as code change in 13.0

Reply:
Done :+1: